core/src/main/java/com/sequenceiq/cloudbreak/service/freeipa/FreeIpaCleanupService.java

```diff
@@ -45,7 +45,7 @@
 
     private static final String ROLE_NAME_PREFIX = "hadoopadminrole-";
 
-    private static final int POLL_INTERVAL = 5000;
+    private static final int POLL_INTERVAL = 2000;
```

Comment:
I wouldn't change this, as it gains only 3 sec at best and increase the polling intensity by 2.5

Reply:
There are not a lot of big rocks in terms of trying to save time for these operations... which is why there's changes to polling intervals on several different operations (roughly based on how long the operations take).
In this case the operation for a single node takes about 3 seconds, and the that guarantees a couple of seconds lost since the first poll is immediate. Multiple nodes, OTOH, can take longer.

Reply:
Reverted to 5 seconds